dll/base.cpp

```diff
@@ -22,6 +22,9 @@
 #include "dbg_log/dbg_log.hpp"
 #endif
 
+#ifdef __LINUX__
+#include <dlfcn.h>
```

Comment:
No need to include header here because it is already included in `common_includes.h`:
https://github.com/Detanup01/gbe_fork/blob/27b7c72901d99da9dfdc7853d16d959d14235bc3/dll/dll/common_includes.h#L136